src/test/java/uk/gov/hmcts/probate/service/NotificationServiceTest.java

```diff
@@ -1201,6 +1255,19 @@ public void sendSolsCaveatRaisedCtscEmail()
         personalisation.put(PERSONALISATION_SOLICITOR_REFERENCE, solicitorCaveatRaisedCaseData.getData().getSolsSolicitorAppReference());
         personalisation.put(PERSONALISATION_CAVEAT_EXPIRY_DATE, "1st January 2019");
         personalisation.put(PERSONALISATION_WELSH_CAVEAT_EXPIRY_DATE, "1 Ionawr 2019");
+        personalisation.put(PERSONALISATION_WELSH_DATE_OF_DEATH, "12 Rhagfyr 2000");
+        personalisation.put(PERSONALISATION_DATE_OF_DEATH, "12th December 2000");
+        if(caveatData.getDeceasedDateOfBirth() != null){
```

Comment:
Do you need both ENG and WEL here? Does the template need both? I would have thought the template only needs one or other - in which case you should be able to determine ENG/WEL from the case data?

Reply:
I need both. The template provides both English and Welsh.